docs/hpc/applications/easybuild.md

```diff
@@ -38,195 +198,249 @@ archspec/0.1.4-GCCcore-11.3.0                            SCOTCH/7.0.1-iimpi-2022
+  1) tools/dev                        12) libevent/2.1.12-GCCcore-13.2.0   23) ScaLAPACK/2.2.0-gompi-2023b-fb  34) cryptography/41.0.5-GCCcore-13.2.0
+  2) GCCcore/13.2.0                    13) UCX/1.15.0-GCCcore-13.2.0        24) foss/2023b                      35) virtualenv/20.24.6-GCCcore-13.2.0
+  3) zlib/1.2.13-GCCcore-13.2.0        14) libfabric/1.19.0-GCCcore-13.2.0  25) bzip2/1.0.8-GCCcore-13.2.0      36) Python-bundle-PyPI/2023.10-GCCcore-13.2.0
+  4) binutils/2.40-GCCcore-13.2.0      15) PMIx/4.2.6-GCCcore-13.2.0        26) ncurses/6.4-GCCcore-13.2.0      37) pybind11/2.11.1-GCCcore-13.2.0
+  5) GCC/13.2.0                        16) UCC/1.2.0-GCCcore-13.2.0         27) libreadline/8.2-GCCcore-13.2.0  38) SciPy-bundle/2023.11-gfbf-2023b
+  6) numactl/2.0.16-GCCcore-13.2.0     17) OpenMPI/4.1.6-GCC-13.2.0         28) Tcl/8.6.13-GCCcore-13.2.0       39) networkx/3.2.1-gfbf-2023b
+  7) XZ/5.4.4-GCCcore-13.2.0           18) OpenBLAS/0.3.24-GCC-13.2.0       29) SQLite/3.43.1-GCCcore-13.2.0    40) mpi4py/3.1.5-gompi-2023b
+  8) libxml2/2.11.5-GCCcore-13.2.0     19) FlexiBLAS/3.3.1-GCC-13.2.0       30) libffi/3.4.4-GCCcore-13.2.0     41) GROMACS/2024.1-foss-2023b
+  9) libpciaccess/0.17-GCCcore-13.2.0  20) FFTW/3.3.10-GCC-13.2.0           31) Python/3.11.5-GCCcore-13.2.0
+ 10) hwloc/2.9.2-GCCcore-13.2.0        21) gompi/2023b                      32) gfbf/2023b
+ 11) OpenSSL/1.1                       22) FFTW.MPI/3.3.10-gompi-2023b      33) cffi/1.15.1-GCCcore-13.2.0
 ```
 
-Note: As evident from the example above, apart from the "gateway" module "tools/dev" only the module for the software you want to use is required to be loaded. Any other module which is required will be loaded now automagically. This is different from the old system where you might need to load a compiler and MPI module. The new system that does not required this as the requirements for that software will be loaded automatically.
-Also, if you want to use two different programs at the same time, say A and B, you need to make sure you are using the same toolchain. In the example above, the toolchain would be foss and the version would be 2021b. Thus, you only could use B/1.2.3-foss-2021b but not B/1.2.3-foss-2022a together with GROMACS/2021.5-foss-2021b example from above!
+Note: As evident from the example above, apart from the "gateway" module`tools/dev` only the module for the software you want to use is required to be loaded. Any other module which is required will be loaded now automagically. It does not print out which modules are loaded, per default `ml` *silently* loads the required modules. Also, only the module of the software you want to use is required, all dependencies are automagically resolved. Furthermore, the `-foss-2023b` is what is called a *toolchain*. You only can load software within the same toolchain hierarchy, you cannot mix-and-match. See [here](https://docs.easybuild.io/common-toolchains/#toolchains_diagram) for more information.
```

Comment:
```suggestion
Note: As evident from the example above, apart from the "gateway" module`tools/dev` only the module for the software you want to use is required to be loaded. Any other module which is required will be loaded now automagically.
```

Reply:
I was trying to address a few things here:
- modules are loaded in the background without echoing which ones are loaded. Some users coming from CX3 were originally confused.
- there is no need to load any other modules apart from the software you want to use. Again, some users coming from CX3 were confused and loaded unrelated modules
- I also wanted to take up the opportunity to remind users about the various toolchains. This is different from CX3 where software was simply compiled with the OS compiler, or a specific Intel compiler which needed to be loaded extra

Reply:
What I meant is that same text has been used in the line 129 and you described it there as well. It is not necessary to repeat things.

Reply:
True, but that is a different section as we discussed. In my opinion, good online user documentation should allow a user to dive in a specific chapter and take it from there, with maybe reading some of the references, without reading the whole documentation from A-Z. 
As we discussed, there are probably more than one approaches of how to write the documentation. As we agreed for now we just leave that as it is.